### DISCUSSION TITLE
从我博客的超链接跳转就一直卡在中转界面，直接输入url却是正常

### DISCUSSION THREAD

**Discussion by libasten** [travellings-link/travellings#1214] [寻求帮助] [answered]
我博客网站左侧侧边栏放了”开往“的超链接：
[https://www.feidaoboke.com](https://www.feidaoboke.com)
博客服务是部署在腾讯香港轻量云上的。

每次点击这个超链接，总是卡顿在中转界面上，F12报错信息如下：

![QQ截图20220923092041](https://user-images.githubusercontent.com/8624509/191876830-94f4342f-6655-4148-8bc0-5ea02e39a3f8.jpg)

如果在我的浏览器中直接输入

https://travellings.link/

则能正常跳转，这是啥原因？

**Comment by Jiaocz** [comment]
在commit [travelling-link/travellings#48f4795](https://github.com/travellings-link/travellings/commit/48f479534e2ce72ded28fbc0ecc57e92484f29ad) 中，文件`/assets/travelling.js` 中的 `url` 被修改为 `urls`，但是 [L634](https://github.com/travellings-link/travellings/blob/48f479534e2ce72ded28fbc0ecc57e92484f29ad/assets/travelling.js#L634) 中一句关于 referer 的 url 没有被替换，才会有从博客跳转时会有问题，但是直接打开没有问题

**Reply by Jiaocz** [reply]
可以转为 Issue 跟踪解决状态

**Reply by libasten** [reply]
> 可以转为 Issue 跟踪解决状态

这是等你们开发人员更新代码吗？我这边不要做什么设置跳转了吧？

**Reply by Jiaocz** [reply]
> > 可以转为 Issue 跟踪解决状态
> 
> 这是等你们开发人员更新代码吗？我这边不要做什么设置跳转了吧？

不需要，等更新就好

**Comment by StarWEB890** [comment]
我这边直接访问会出现你类似的问题
只不过我已经提过一次Discussion了：#1205

**Comment by Jiaocz** [comment, marked as answer]
在 #1218 解决了，等后续部署后就没有问题了
